### PR TITLE
Type annotations in Display_text label library as a sample test for pylint and CI verifications

### PR DESCRIPTION
**Pull request by jposada202020**
As discussed during Saturday stream 2020-02-27 we want to include type annotations inside libraries code. Using this sample we want to verify CI behavior and possible solutions.

### CODE REVIEW

**Comment by jposada202020** [comment]
As discussed: @hugodahl. Thanks let me know if you need help with anything :).
Nothing new here is falling since last week for the pylint bug. Thanks. Please review and comment.

File: adafruit_display_text/label.py [hunk 1]
```diff
@@ -140,7 +140,7 @@ def __init__(
         if (anchored_position is not None) and (anchor_point is not None):
             self.anchored_position = anchored_position
 
-    def _create_background_box(self, lines, y_offset):
+    def _create_background_box(self, lines: int, y_offset: int):
```
Comment (hugodahl) [inline adafruit_display_text/label.py:143]: Looks like one missed ` -> None` for the return type. All other locations already appear to have it added

File: adafruit_display_text/label.py [hunk 2]
```diff
@@ -457,7 +456,7 @@ def anchored_position(self):
         )
 
     @anchored_position.setter
-    def anchored_position(self, new_position):
+    def anchored_position(self, new_position: Tuple[int, int]) -> Tuple[int, int]:
```
Comment (hugodahl) [inline adafruit_display_text/label.py:459]: Since this is the setter for the `anchored_position` property, shouldn't the return type hint be `None`?

**Review by hugodahl — COMMENTED** [review]
Just those two minor type-hint issues.

File: adafruit_display_text/label.py (hunk 2 above)
Comment (jposada202020) [reply adafruit_display_text/label.py]: Thanks for the revision, you are right!

File: adafruit_display_text/label.py (hunk 1 above)
Comment (jposada202020) [reply adafruit_display_text/label.py]: Yes, I miss that one, thanks for pointing that out!

**Comment by jposada202020** [comment]
@hugodahl Thanks for the revision I made all the changes as suggested thanks for catching those up!!!

**Comment by FoamyGuy** [comment]
This PR was superseded by #134 the type hints were included in that PR. The Tab replacement fix code was included with #132 

Thanks for working on this @jposada202020!

**Pull request closed without merging**